From 493d521d9ffe706741665a88ea14929913ea2eaf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 23 Sep 2014 09:22:40 -0400
Subject: [PATCH] Fix warning about unused variable with !SELINUX

src/shared/label.c:255:15: warning: unused variable 'l' [-Wunused-variable]
         char *l = NULL;
               ^
---
 src/shared/label.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/shared/label.c b/src/shared/label.c
index 02b41f02d8..b6af38d82d 100644
--- a/src/shared/label.c
+++ b/src/shared/label.c
@@ -252,9 +252,10 @@ fail:
 
 int label_get_our_label(char **label) {
         int r = -EOPNOTSUPP;
-        char *l = NULL;
 
 #ifdef HAVE_SELINUX
+        char *l = NULL;
+
         r = getcon(&l);
         if (r < 0)
                 return r;
